blaze-core/src/test/scala/org/http4s/blaze/util/EntityBodyWriterSpec.scala

```diff
@@ -100,92 +99,106 @@ class ProcessWriterSpec extends Http4sSpec {
     runNonChunkedTests(tail => new CachingChunkWriter(new StringWriter(), tail, Task.now(Headers())))
   }
 
-  "ChunkProcessWriter" should {
+  "ChunkEntityBodyWriter" should {
     def builder(tail: TailStage[ByteBuffer]) =
-      new ChunkProcessWriter(new StringWriter(), tail, Task.now(Headers()))
-
-    "Not be fooled by zero length chunks" in {
```

Comment:
Is there a new version of this test, or is it no longer possible for some reason? This can really mess up a `Transfer-Encoding: chunked` response.